sources/dataset.py

```diff
@@ -32,15 +32,12 @@
 
 import utils
 
-
 class Dataset(object):
-
-    def __init__(self, image_size, glossary_filename):
-        """ Class constructor
-        """
+    """
+    """
+    def __init__(self, image_size):
         self.image_size = image_size
         self.class_info = defaultdict()
```

Comment:
I don't understand the usage of two defaultdict for `class_info` and `image_info` attributes. Classic `dicts` such as `self.class_info = {}` are not enough?

Reply:
Defaultdicts have been turned into simple dicts.